sim/src/main.rs

```diff
@@ -9,33 +9,35 @@ mod traffic_generator;
 
 use channel::Channel;
 use link::Link;
-use plugin_wrapper::PluginWrapper;
 use simulator::Simulator;
 use traffic_generator::TrafficGenerator;
-use std::borrow::BorrowMut;
 use std::path::PathBuf;
 
-static COMPILED: &str = "target/debug/libfilter_lib";
+//static COMPILED: &str = "../target/debug/libfilter_lib";
```

Comment:
Remove commented out code if it is not needed. If there is a reason, provide a comment.